using-mobile-extensions/adobe-audience-manager/README.md

```diff
@@ -49,27 +51,11 @@ To get started with **Audience Manager**, follow these steps:
    import ACPAudience
   ```
 
-   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing manually, ensure that you have also added the `ACPIdentity.a` to your project.
+   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing the Audience Manager Extension manually, ensure that you have also added the `libACPIdentity_iOS.a` to your project.
```

Comment:
- In "When installing the Audience Manager Extension manually", please change "Extension" to "extension".

- In "ensure that you have added the `identity-1.x.x.aar` file to your project.", please remove "have".